NTO/Automation/entities/KnowledgeItem.ttl

```diff
@@ -76,5 +76,6 @@ In that case, the KnowledgeItem executes operations, which serves the exploitati
 			[ ogit:connects  ogit.Automation:KnowledgeItem ]
 			[ ogit:connects  ogit.Forum:Post ]
 			[ ogit:connects  ogit:Hyperlink ]
+			[ ogit:contains  ogit:Tag ]
```

Comment:
I would question semantics here... a KnowledgeItem does not "contain" a tag but a Tag "describes" or "qualifies" a KnowledgeItem, doesn't it?

Reply:
Fair point, I'll change this

Reply:
I have removed the relationship from the Knowledge Item and have included `ogit:Tag ogit:describes ogit:Automation:KnowledgeItem`

Reply:
I have removed the edge from tag altogether now